senders/victorops/api/alert.go

```diff
@@ -0,0 +1,63 @@
+package api
+
+import (
+	"bytes"
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"net/http"
+)
+
+// CreateAlertRequest the API request to be made to
+// create a victorops alert
+type CreateAlertRequest struct {
```

Comment:
There are a lot of additional fields in [grafana integration](https://github.com/grafana/grafana/blob/d8736a25475d146bd7111bddc5739e1380a1b0d3/pkg/services/alerting/notifiers/victorops.go#L104) and in a [third-party client](https://github.com/chrissnell/victorops-go/blob/master/client.go#L17), let's start transferring them too.

Reply:
I would pass these fields:
- timestamp: time.Now().Unix()
- entity_id: TriggerID (It seemed to me that this field is required. Have you checked if the integration works?)
- monitoring_tool: Moira